Lib/test/test_inspect.py

```diff
@@ -1136,6 +1136,18 @@ class C(metaclass=M):
         attrs = [a[0] for a in inspect.getmembers(C)]
         self.assertNotIn('missing', attrs)
 
+    def test_getmembers_with_module(self):
```

Comment:
I think it would be better to change the name (and perhaps the code) of the test. There is nothing specific for modules. Instead we need to test that the instance attribute `__bases__` doesn't break `inspect.getmembers()`.